TravelAgency-service/src/test/java/cz/muni/fi/travelAgency/facade/ExcursionFacadeTest.java

```diff
@@ -128,14 +129,18 @@ public void setUp() {
      */
     @Test
     public void createExcursionTest() {
-        Mockito.doAnswer(invocationOnMock -> {
-            Object argument = invocationOnMock.getArgument(0);
-            assert (argument instanceof Excursion);
-            ((Excursion) argument).setId(excursionKutaBeach.getId());
-            return null;
-        }).when(excursionService).createExcursion(Mockito.any(Excursion.class));
-        excursionFacade.createExcursion(excursionKutaBeachDTO);
-        Assert.assertEquals(excursionKutaBeachDTO.getId(), excursionKutaBeach.getId());
+//        Mockito.doAnswer(invocationOnMock -> {
```

Comment:
By using custom DTOs for all the operations you created a lot of work for us/yourself :D Is it really necessary to use them ? I think it's not needed. Why did you choose to implement it ?